modules/gke-cluster-standard/main.tf

```diff
@@ -419,13 +419,24 @@ resource "google_gke_backup_backup_plan" "backup_plan" {
       }
     }
 
-    all_namespaces = lookup(each.value, "namespaces", null) != null ? null : true
+    all_namespaces = each.value.all_namespaces ? true : null
```

Comment:
This because onyl one of _all_namespaces_, _selected_namespaces_ and _selected_applications_ can be defined.